decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -167,6 +175,18 @@ namespace :decidim do
     def system!(command)
       system("cd #{rails_app_path} && #{command}") || abort("\n== Command #{command} failed ==")
     end
+
+    def tailwind_variables
+      # Get the list of Decidim gems installed and adds the current folder ('.')
+      content_directories = Dir.glob("#{Gem.loaded_specs["decidim"].full_gem_path}/decidim-*").push(".")
```

Comment:
If you are writing these paths to the static configuration file, you will face issues when you move the same instance to another environment, e.g. from dev to staging/production. Therefore, I would not statically write these to the configuration files as they depend on the environment.

Also, how are now modules added to the configuration paths? As I understood this is only run during `decidim:webpacker:install`.

Reply:
We discussed this today in the maintainers meeting and the conclusion was:
1. We'll merge this PR first to the `redesign` branch
2. The redesign team will think of a more dynamic way to do this as the work progresses

A reference point to this is something similar what we did with Webpacker which generates the list of paths dynamically when before running webpack and stores this list in the `tmp` folder for the JS to access that file.

Reply:
I've added it as a note in the project, so we don't miss it. https://github.com/decidim/decidim/projects/23#card-82995506

Reply:
PR merged with dynamic configuration generated on runtime, so closing the note.